pages.pt-BR/common/7z.md

```diff
@@ -5,23 +5,23 @@
 
 - Compactar um arquivo ou diretório:
 
-`7z a {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretorio}}`
+`7z a {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
 
 - Criptografar um arquivo existente (incluindo cabeçalhos):
 
-`7z a {{arquivo_criptografado.7z}} -p {{senha}} -mhe {{arquivo.7z}}`
+`7z a {{arquivo_criptografado.7z}} -p{{senha}} -mhe {{arquivo.7z}}`
```

Comment:
Maybe here it makes sense to split -mh from -e, and remove the space between -e and the filename? Or is it unrelated?

Reply:
The man page isn't clear, but I think that `-mhe` is all one option.

Reply:
Wow, really? These pages seem to have quite cryptic interface :/

Reply:
`-mhe` is the `-m` option with parameter `he`, as documented in the man page: 

```-m{Parameters}
-mhe=on|off
7z format only : enables or disables archive header encryption (Default : off)```

Reply:
Hey guys, I found this:

> he=[off | on]
> Enables or disables archive header encryption. The default mode is he=off.

I looked the file (/usr/share/doc/p7zip/DOC/MANUAL/cmdline/switches/method.htm) suggested on manual.

Reply:
I'm tempted to suggest `-m"he"` or `-m'he'`, just to be sure this is properly understood. I suppose that'd work, right?

Reply:
It's the same for me really, I don't have a preference, both look clear enough.

Reply:
@andrik what do you think?

Reply:
I think it's better using -mhe=on

Reply:
May I change?

Reply:
> May I change?

No objections to the change were raised, so please go ahead :)

Reply:
Done